src/test/java/com/rocketden/main/util/SocketTestMethods.java

```diff
@@ -70,7 +70,7 @@ public static void createSingleProblemAndTestCases(TestRestTemplate template, in
         createProblemRequest.setName(NAME);
         createProblemRequest.setDescription(DESCRIPTION);
         createProblemRequest.setDifficulty(ProblemDifficulty.EASY);
-
+        createProblemRequest.setApproval(true);
```

Comment:
nit: When you add lines, thoughts on keeping these line spacers between different segments of the code? There are a good number of other places with this as well, if you're open to it you could add those blank lines between the different code sections.

Reply:
Will definitely keep this in mind for future PRs! I looked through some of the files I modified and couldn't find many instances of this, though I'm 100% sure they exist; sorry about that

Reply:
No problem, all good 👍